# Created on 2022-01-30T09:26:01.160
org.testng.TestNGException: 
Two tests in the same suite [ContinousTestingAddressbook] cannot have the same name: Test1
	at org.testng.xml.internal.XmlSuiteUtils.ensureNoDuplicateTestsArePresent(XmlSuiteUtils.java:65)
	at org.testng.xml.internal.XmlSuiteUtils.validateIfSuitesContainDuplicateTests(XmlSuiteUtils.java:30)
	at org.testng.TestNG.sanityCheck(TestNG.java:981)
	at org.testng.TestNG.run(TestNG.java:1010)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:379)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:340)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:413)


